src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE_NUMBER = 0;
+    private static final int OPERATION_VALUE_NUMBER = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(createReport(readFile(fromFileName)), toFileName);
+    }
+
+    private String[] readFile(String fileName) {
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fileName));
+            String value = reader.readLine();
+            StringBuilder stringBuilder = new StringBuilder();
+            while (value != null) {
+                stringBuilder.append(value).append(System.lineSeparator());
+                value = reader.readLine();
+            }
+            return stringBuilder.toString().split(System.lineSeparator());
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException("Can't find file!", e);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private String createReport(String[] linesFromFile) {
```

Comment:
This method doing 2 things:
  1. calculation suppy sum and buy sum
  2. creating report string
I think it will be better to separate this responsibilities